src/protocol/models.ts

```diff
@@ -36,14 +34,14 @@ export interface AuthorizationRequestBody {
   reason: string;
   callbackUrl: string;
   scope: ZKPRequest[];
-  did_doc?: DIDDocument;
+  did_doc?: Uint8Array;
```

Comment:
why

Reply:
Since golang implementation also uses `[]byte`

Reply:
did_doc nowhere uses

Reply:
it's not a reason. I'd like to have a DIDDocument interface. 
in golang it's Uint8array because it can't parse nested structures and types.

Reply:
fixed